run/README.namelist

```diff
@@ -942,6 +947,13 @@ Namelist variables for controlling the adaptive time step option:
                                                   28 - for USGS if including lake category
                                                   21 - for MODIS if including lake category (default since 3.8)
 						  40 - for NCLD
+ use_wudapt_lcz			     = 0,       ! Option to use WUDAPT LCZ urban landuse categories, along with standard
```

Comment:
@kkeene44 Can you line these lines up?

Reply:
@weiwangncar 
I assume you were adding all of these "line up" comments as I was correcting them. They should all be okay now.